src/Components/Artwork/GridItem.tsx

```diff
@@ -28,11 +28,52 @@ interface Props extends RelayProps, React.HTMLProps<ArtworkGridItemContainer> {
   }
 }
 
-class ArtworkGridItemContainer extends React.Component<Props, null> {
+interface State {
+  width: number
+  height: number
+}
+
+const GEMINI_HOST = "d7hftxdivxxvm.cloudfront.net"
```

Comment:
We should probably be reading this from ENV, maybe that can be a follow up `FIXME`?